packages/nouns-webapp/src/components/NavBarButton/index.tsx

```diff
@@ -0,0 +1,58 @@
+import classes from './NavBarButton.module.css';
+
+interface NavBarButtonProps {
+    buttonText: string;
+    buttonIcon?: React.ReactNode;
+    buttonStyle?: string;
+}
+
+const NavBarButton: React.FC<NavBarButtonProps> = props => {
+    const { buttonText, buttonIcon, buttonStyle } = props;
+
+    const variant = () => {
```

Comment:
Probably a good idea here to use enums here instead of stand alone strings so that when using the button styles outside of the `NavBarButton` we can import the enum and see the options instead of having to dig into the `NavBarButton` file for the possible styles.

Reply:
I updated this to use enums. Let me know if this was what you had in mind.

Reply:
looks great! this is exactly what i had in mind.